src/site_assessment/common_functions.jl

```diff
@@ -159,15 +159,23 @@ end
         search_buffer::Union{Int64,Float64}=20000.0
     )::Float64
 
-Identifies the closest edge to the target `pixel`/'`geom_buff` and returns the initial rotation
-angle required to match the edge line.
+Identifies the closest edge to the target `pixel`/'`geom_buff`. The angle required to rotate
+geom_buff by to match this reef edge is calculated.
 
 # Arguments
 - `pixel` : Target point at the center of the search polygon.
 - `geom_buff` : Initial search box with zero rotation.
 - `gdf` : GeoDataFrame containing a geometry column used for pixel masking.
 - `reef_outlines` : Line segments for the outlines of each reef in `gdf`.
 - `search_buffer` : Distance to search from pixel to find closest reef.
+
+# Returns
+Rotation angle required to match reef edge when used in `rotate_geom(geom_buff, rot_angle)`.
+
+# Extended help
+The returned angle is the angle relative to the default `geom_buff` horizontal orientation.
+Therefore, if returned angle = 45 degrees, `rot_geom(geom_buff, 45)` will rotate `geom_buff`
+by 45 degrees to match the identified reef edge.
```

Comment:
Extended help goes below the main description text.

The `# Returns` section is always the last section.